<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Операнд 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Операнд 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="operand1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="operand2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="operations">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>211</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Операции</string>
    </property>
    <widget class="QRadioButton" name="delenie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Деление</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="umnozhenie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Умножение</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="slozhenie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Сложение</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="vichitanie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Вычитание</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="sin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sin</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cos">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cos</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="tg">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tg</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ctg">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ctg</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="kvadrat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Квадрат</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="koren">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Корень</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="asin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ASin</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="acos">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ACos</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="atg">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ATg</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="actg">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ACtg</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="res">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultat">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
